### DISCUSSION TITLE
主页的 定制主题，随心所欲 那个示例 能否 开源一下。

### DISCUSSION THREAD

**Discussion by CHENJKNB** [ant-design/ant-design#39544] [Q&A]
1. **大佬们 这个页面 能否开源一下。**

![image](https://user-images.githubusercontent.com/18523983/207477937-0eca4d89-e07c-4d58-95d2-c2c58ad873f4.png)

**Comment by yoyo837** [comment]
在源码里面呀

**Comment by zombieJ** [comment]
https://github.com/ant-design/ant-design/blob/master/.dumi/pages/index/components/Theme/index.tsx
